tracer/test/snapshots/OpenTelemetrySdkTests.verified.txt

```diff
@@ -949,6 +949,33 @@ at Samples.OpenTelemetrySdk.Program.RunSpanUpdateMethods(TelemetrySpan span),
     TraceId: Id_40,
     SpanId: Id_41,
     Name: internal,
+    Resource: service.name should be the DefaultServiceName value,
```

Comment:
I  really like this approach of having the verbose "assertion" in the Resource to make it easier to check the spans 👍